models/board.js

```diff
@@ -0,0 +1,41 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+const User = require('./User');
+
+// create Board model
+class Board extends Model { }
+
+// define table columns and configuration
+Board.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    // board_content: {
+    //   type: DataTypes.JSON,
+    //   allowNull: false,
+    // },
+    user_id: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'user',
+        key: 'id'
+      }
+    }
+  },
+  {
+    sequelize,
+    freezeTableName: true,
+    underscored: true,
+    modelName: 'board'
+  }
+);
+
+module.exports = Board;
```

Comment:
New line at ends of files. I think this can be set in eslint as an option. Do you think we should include that in the project?

Reply:
I think I updated my VS code settings to include the new line. Can you check in the next push to see if it worked?